staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -73,7 +73,9 @@ type ListMeta struct {
 
 // These are internal finalizer values for Kubernetes-like APIs, must be qualified name unless defined here
 const (
-	FinalizerOrphan string = "orphan"
+	FinalizerOrphan           string = "orphan"
+	FinalizerOrphanDependents string = "orphan"
+	FinalizerDeleteDependents string = "foregroundDeletion"
```

Comment:
I'd lean towards consistency and leave it for now, since we already have orphan, and therefore will have to do a name change in the future anyway. It should probably have been `kubernetes.io/garbagecollector=foreground`, though. @bgrant0607 may disagree with me.